internal/tests/cli/boundary/workers.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+load _auth
+load _workers
+load _helpers
+
+@test "boundary/login: can login as default user" {
+  run login $DEFAULT_LOGIN
+  echo "$output"
+  [ "$status" -eq 0 ]
+}
+
+# TODO: Add worker creation and deletion tests.
+
+@test "boundary/workers: can list workers" {
+	run list_workers
+  echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "boundary/workers: can update worker's name" {
+  local wid=$(worker_id)
+  local name="test"
+    run update_worker $wid $name
+  echo "$output"
+	[ "$status" -eq 0 ]
```

Comment:
It looks like this is failing with "cannot update a KMS worker"

Reply:
I didn't run it locally- it failed on Enos: https://github.com/hashicorp/boundary/runs/7982789871?check_suite_focus=true